src/main/java/com/microsoft/azure/functions/worker/handler/WorkerInitRequestHandler.java

```diff
@@ -18,6 +18,16 @@ String execute(WorkerInitRequest request, WorkerInitResponse.Builder response) {
         response.putCapabilities("WorkerStatus", "WorkerStatus");
         response.putCapabilities("RpcHttpBodyOnly", "RpcHttpBodyOnly");
         response.putCapabilities("RpcHttpTriggerMetadataRemoved", "RpcHttpTriggerMetadataRemoved");
+        response.setWorkerMetadata(composeWorkerMetaData());
         return "Worker initialized";
     }
+
+    private WorkerMetadata.Builder composeWorkerMetaData(){
```

Comment:
super nit: metadata is one word 
```suggestion
    private WorkerMetadata.Builder composeWorkerMetadata(){
```